core/src/main/java/com/graphhopper/routing/lm/LMAlgoFactoryDecorator.java

```diff
@@ -0,0 +1,215 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for 
+ *  additional information regarding copyright ownership.
+ * 
+ *  GraphHopper GmbH licenses this file to you under the Apache License, 
+ *  Version 2.0 (the "License"); you may not use this file except in 
+ *  compliance with the License. You may obtain a copy of the License at
+ * 
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.routing.lm;
+
+import com.graphhopper.routing.AlgorithmOptions;
+import com.graphhopper.routing.RoutingAlgorithm;
+import com.graphhopper.routing.RoutingAlgorithmFactory;
+import com.graphhopper.routing.RoutingAlgorithmFactoryDecorator;
+import com.graphhopper.routing.util.HintsMap;
+import com.graphhopper.routing.util.TraversalMode;
+import com.graphhopper.routing.weighting.Weighting;
+import com.graphhopper.storage.Graph;
+import com.graphhopper.storage.GraphHopperStorage;
+import com.graphhopper.util.CmdArgs;
+import com.graphhopper.util.PMap;
+import com.graphhopper.util.Parameters.Landmark;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * This class implements the A*, landmark and triangulation (ALT) decorator.
+ *
+ * @author Peter Karich
+ */
+public class LMAlgoFactoryDecorator implements RoutingAlgorithmFactoryDecorator {
+    private int landmarkCount = 8;
+    private int activeLandmarkCount = landmarkCount / 2;
```

Comment:
No, this does not matter. But I'll make the defaults a bit different as tests show that 8 active landmarks are nearly always the best configuration despite the total number of landmarks